caesar.c

```diff
@@ -0,0 +1,120 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include "caesar.h"
+
+//Function Declarations
+int _CheckChar(char chr);
+int _CheckNumOfShifts(int num, int* ALPshift, int* DGshift);
+void _EncodeChar(char* add_char, int shiftDirect, int modValue, int numShift, int arraySize);
+
+//Function Definitions 
+Input GetInputStringInfo(char* string)
+{
+	Input stringInfo;
+	//Get number of shifts
+	stringInfo.numShift = atoi(strtok(string, ":"));
+	
+	//Get the actual Input string and malloc it 
+	char* actstring = strtok(NULL, "\0");
```

Comment:
Prefer verbose variable names or use common abbreviations. I.e. "actualString" is more readable